README.md

```diff
@@ -362,6 +362,42 @@ The `SKIPPED` result code indicates that a given test was explicitly skipped by
 * taking a very long time to run
 * failing or passing for the incorrect reason
 
+### Debugging individual tests
+
+While `stl-lit` is super awesome in finding out that *something* is wrong or not even compiling, it is not really
+helpful in debugging *what* is going wrong. However, debugging individual tests is rather simple given some additional
+steps. Let's assume we want to debug a new feature with tests located in `tests\std\tests\GH_XXXX_meow`
+
+As always, build the STL from your branch and run the tests:
+```
+C:\STL\out\build\x64> ninja
+C:\STL\out\build\x64> python tests\utils\stl-lit\stl-lit.py -v C:\STL\tests\std\tests\GH_XXXX_meow
+```
+
+Let's assume one of the tests fails an assert and we want to debug that configuration. `stl-lit` will conveniently print
+the build command, which is far too long to provide here in full. The important part is to add the following option to
+provide debug symbols: `/Zi /Fdbark.pdb`.
+
+You can replace `bark` with any descriptive name you like. Add these before the `"-link"` option in the command line
+and recompile.
+Example:
+```
+C:\STL\out\build\x64>"C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\VC\Tools\MSVC\14.29.29917\bin\HostX64\x64\cl.EXE" "C:\STL\tests\std\tests\GH_XXXX_meow\test.cpp" "-IC:\STL\out\build\x64\out\inc" "-IC:\STL\llvm-project\libcxx\test\support" "-IC:\STL\tests\std\include" "/nologo" *SUPER_IMPORTANT_BUILD_CONFIGURATIONS*"
+"-FeC:\STL\out\build\x64\tests\std\tests\GH_XXXX_meow\Output\02\GH_XXXX_meow.exe" /Zi /Fdbark.pdb "-link" "-LIBPATH:C:\STL\out\build\x64\out\lib\amd64" "-LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\VC\Tools\MSVC\14.29.29917\lib\x64" "/MANIFEST:EMBED"
+```
+
+You can now start debugging the test via
+```
+devenv "C:\STL\out\build\x64\tests\std\tests\GH_XXXX_meow\Output\02\GH_XXXX_meow.exe" "C:\STL\tests\std\tests\GH_XXXX_meow\test.cpp"
+```
+
+However, this will not work right away, as Visual Studio will complain about a missing `msvcp140_oss.dll`. The reason is

```

Comment:
This extra step is only necessary if the executable to debug was dynamically linked (`/MD` or `/MDd`). Maybe point out that you can ignore this step if it is statically linked (`/MT` or `/MTd`)?

Reply:
That would require to actually parse the command line for the right arguments, whereas if we add it to the path it will always work.

Reply:
I rephrased the sentence to get the best of both worlds

Reply:
> That would require to actually parse the command line for the right arguments, whereas if we add it to the path it will always work.

True, but I was bothered by the unconditional nature of "this will not work right away", which is incorrect when the exe is statically linked.

> I rephrased the sentence to get the best of both worlds

LGTM.